services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/AlluClient.kt

```diff
@@ -206,6 +209,18 @@ class AlluClient(
         }
     }
 
+    fun operationalCondition(alluApplicationId: Int, operationalConditionDate: LocalDate) {
```

Comment:
Other methods have INFO logging at the beginning - should this too?

Reply:
Yes, I added the logging.